argilla/mkdocs.yml

```
@@ -191,6 +191,7 @@ nav:
   - Community:
       - community/index.md
       - How to contribute?: community/contributor.md
+      - Add a new language to Argilla: community/adding_language.md
       - Developer documentation: community/developer.md
```

Maybe it makes more sense after the developer documentation?

```suggestion
      - Developer documentation: community/developer.md
      - Add a new language to Argilla: community/adding_language.md
```

---

Yes absolutly